resources/js/lang/en.js

```diff
@@ -48,19 +46,12 @@ export default {
             },
         },
-        5: {
-            "name": "Scry Me a River",
-            "stages": {
-                0: "Jekserah has fled and her whereabouts are unknown. The Aesther enchantress Hail may know of a way to divine her location through scrying.",
-                1: "Jekserah has fled and her whereabouts are unknown. The Aesther enchantress Hail at the Crooked Bone may know of a way to divine her location through scrying.{br}Hail sent you to collect bitteroot from the Dagger Forest. The absent-minded enchantress then used the bitter root to divine Jeksersah's location. Also, you now know to ring the bell and wait 47 seconds before entering the Crooked Bone."
-            },
-        },
         6: {
             "name": "The Merchant's Secret",
             "stages": {
-                0: "A mysterious contact has given you the location of Jekserah's warehouse in Gloomhaven. Jekserah is clearly hiding a secret in this warehouse. Raiding the warehouse will not bode well for any open contracts you have with her. However, she's definitely planning something and you will put an end to it.",
-                1: "A mysterious contact has given you the location of Jekserah's warehouse in Gloomhaven. Jekserah is clearly hiding a secret in this warehouse. Raiding the warehouse will not bode well for any open contracts you have with her. However, she's definitely planning something and you will put an end to it.{br}The warehouse was full of the undead. Jekserah had clearly been using the necromantic scroll you delivered to raise the undead within the city walls. You fought your way to the back room of the warehouse and confronted Jekserah. She vowed vengeance and flipped out a window while you battled her two Inox bodyguards. After they lay dead, you scoured for evidence of the fleeing Valrath but found none. You’ve stopped her plans for now, but after your brief conversation, it is clear you need to hunt her down before she can make good on her threats.",
-                2: "A mysterious contact has given you the location of Jekserah's warehouse in Gloomhaven. Jekserah is clearly hiding a secret in this warehouse. Raiding the warehouse will not bode well for any open contracts you have with her. However, she's definitely planning something and you will put an end to it.{br}You ignored the city guard Argeise's warnings and continued working for Jekserah. \"Sinister machinations\" is, of course, a troubling phrase to throw around, but the phrase \"considerable reward\" trumps that in a heartbeat. There is no more reason to raid the warehouse."
```

Comment:
This line isn't an exact quote from the previous conclusion and was to me a bit confusing / unnecessary. 
>\"Sinister machinations\" is, of course, a troubling phrase to throw around, but the phrase \"considerable reward\" trumps that in a heartbeat.
It didn't add real info for the blocked state of the Warehouse scenario.

Reply:
It is a direct quote from the scenario #9 intro.
It doesn't add info, but flavor and I'd vote to keep it.

Reply:
This text applies to scenario 8 and is about the preceding events before 8 or 9. If you play scenario 8 you'll block 9 and won't ever read this line (that's what happend in our party)

I'm more concerned about spoilers than flavor. Do you agree that it does spoil a bit?
And I prefer brevity above long descriptions, but that's personal.

Reply:
I don't understand. 
The "Sinister Machinations" text (stage 2) should only be shown if you  have completed 9, and represents one conclusion of this quest.
The other conclusion of this quest is stage 1, which shows if you decide to do scenario 8.

Reply:
You are right. I completely missed that line in scenario 9. I agree that it's better to restore that last part of stage 2.

Reply:
I've restored this with a new commit